scripts/sct_extract_metric.py

```diff
@@ -50,36 +50,25 @@ def get_parser():
 
     param_default = Param()
 
-    # Read .txt files referencing the labels (for extended usage description)
-    file_label = os.path.join(param_default.path_label,
-                              param_default.file_info_label)
-    check_file_exist(file_label, 0)
-    default_info_label = open(file_label, 'r')
-    label_references = default_info_label.read()
-    default_info_label.close()
-
-    description = (f"This program extracts metrics (e.g., DTI or MTR) within labels. Labels could be a single file or "
-                   f"a folder generated with 'sct_warp_template' and containing multiple label files and a label "
-                   f"description file (info_label.txt). The labels should be in the same space coordinates as the "
-                   f"input image.\n"
-                   f"\n"
-                   f"To list white matter atlas labels: {os.path.basename(__file__)} -f "
-                   f"{os.path.join(__data_dir__, 'atlas')}\n"
-                   f"\n"
-                   f"To compute FA within labels 0, 2 and 3 within vertebral levels C2 to C7 using binary method: "
-                   f"{os.path.basename(__file__)} -i dti_FA.nii.gz -f label/atlas -l 0,2,3 -v 2:7 -m bin\n")
-
-    if label_references != '':
-        description += (f"\nTo compute average MTR in a region defined by a single label file (could be binary or 0-1 "
-                        f"weighted mask) between slices 1 and 4: {os.path.basename(__file__)} -i mtr.nii.gz -f "
-                        f"my_mask.nii.gz -z 1:4 -m wa\n"
-                        f"List of labels in {file_label}:\n"
-                        f"--------------------------------------------------------------------------------------\n"
-                        f"{label_references}\n"
-                        f"--------------------------------------------------------------------------------------\n")
-
     parser = argparse.ArgumentParser(
-        description=description,
+        description=(
+            f"This program extracts metrics (e.g., DTI or MTR) within labels. Labels could be a single file or "
+            f"a folder generated with 'sct_warp_template' containing multiple label files and a label "
+            f"description file (info_label.txt). The labels should be in the same space coordinates as the "
+            f"input image.\n"
+            f"\n"
+            f"The labels used by default are taken from the PAM50 template. To learn about the available PAM50 "
+            f"white/grey matter atlas labels and their corresponding ID values, please refer to: "
+            f"https://spinalcordtoolbox.com/en/latest/overview/concepts/pam50.html#white-and-grey-matter-atlas-pam50-atlas\n"
```

Comment:
On second glance, I don't think this is right to say. I think I might have gotten confused, @jcohenadad.

I thought we were using PAM50 by default:

https://github.com/neuropoly/spinalcordtoolbox/blob/5a23e7df5d9b99a661d8a74b684330fdcfe9b8d0/scripts/sct_extract_metric.py#L35-L38

But `Param.path_label` never gets used. Instead, `label/atlas` is used by default:

https://github.com/neuropoly/spinalcordtoolbox/blob/5a23e7df5d9b99a661d8a74b684330fdcfe9b8d0/scripts/sct_extract_metric.py#L102-L108

So pointing users to the PAM50 documentation page is misleading.

Reply:
I think I have an idea for how to redo this PR to handle both #2934 and #3003.

Reply:
> I thought we were using PAM50 by default:

we are: the folder `./label/` is generated by `sct_warp_template`, and uses the PAM50 by default. But if users point to another template than PAM50 as input of `sct_warp_template`, then the info_label.txt will not be the one shown on the website (it represents <1% of our users).

Reply:
In that case, saying "sct_extract_metric uses PAM50 by default" is not _technically_ correct, hehe. I think it might be more informative for the user to say something like:

```
By default, 'sct_extract_metric' will attempt to use the 'label/' directory generated by 
'sct_warp_template'. This means that in most cases, the labels available for metric calculation 
will be the same as the ones found in the PAM50 template. (See also:
https://spinalcordtoolbox.com/en/latest/overview/concepts/pam50.html#white-and-grey-matter-atlas-pam50-atlas)

However, if you chose a different template during the 'sct_warp_template' stage, or 
if you specified a different value for '-f', then the available labels will also be different. 
To double check which labels are available, use the '-list-labels' option alongside your usual 
command to list the labels being used by 'sct_extract_metric'.
```